sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -175,33 +176,16 @@ def binary_subscr(state, unused_arg):
 binary_xor = inplace_xor = symmetric_binary_op
 binary_or = inpalce_or = symmetric_binary_op
 
-# As far as types are concerned.
-slice_0 = nop
```

Comment:
Are these unused?

Reply:
These are not matched, since the opcode name has a `+` instead of `_` (e.g. `SLICE+0`). Also, these were removed in Python 3 so I'd rather not support them.